docs/source/models.rst

```diff
@@ -4114,7 +4211,7 @@ Let's examine the custom flavor module in more detail. The first step is to impo
 inluding ``sktime`` library, various MLflow utilities as well as the MLflow ``pyfunc`` module which
 is required to add the ``pyfunc`` specification to the MLflow model configuration. Note also the
 import of the ``flavor`` module itself. This will be passed to the
-:py:func:`mlflow.models.Model.log()` method to log the model as an artifact to the current MLflow
+:py:func:`mlflow.models.Model.log()` method to log the model as an artifact to the current Mlflow
```

Comment:
In public documentation, I think `MLflow` is recommended, while `Mlflow` and `MLFlow` are not.

Reply:
I think this was changed by the autoformatter, I'll change it back